ocean_lib/ocean/test/test_ocean_assets.py

```diff
@@ -63,18 +63,25 @@ def _get_num_assets(_minter):
     assert ddo, "ddo is not found in cache."
     ddo_dict = ddo.as_dictionary()
     original = original_ddo.as_dictionary()
-    assert ddo_dict["publicKey"] == original["publicKey"]
-    assert ddo_dict["authentication"] == original["authentication"]
-    assert ddo_dict["service"]
-    assert original["service"]
+    assert ddo_dict["publicKey"] == original["publicKey"], (
+        "The new asset does not have the same public key as the " "original asset. "
+    )
+    assert ddo_dict["authentication"] == original["authentication"], (
+        "The new asset does not have the same "
+        "authentication key as the original asset. "
+    )
+    assert ddo_dict["service"], "The new asset does not have the service field."
+    assert original["service"], "The original asset does not have the service field."
     metadata = ddo_dict["service"][0]["attributes"]
     if "datePublished" in metadata["main"]:
         metadata["main"].pop("datePublished")
     assert (
         ddo_dict["service"][0]["attributes"]["main"]["name"]
         == original["service"][0]["attributes"]["main"]["name"]
+    ), ("The new asset does not have the same name as the " "original asset. ")
```

Comment:
here: concatenation and simplification.
I'd suggest: "The new asset has a different name.", "The new asset has different access service details."